### DISCUSSION TITLE
Vite.js support in CodeSandbox

### DISCUSSION THREAD

**Discussion by MatejBransky** [codesandbox/codesandbox-client#4671] [General]
Hi, is it possible to support importing [Vite.js](https://github.com/vitejs/vite) based projects from github in CodeSandbox (https://codesandbox.io/docs/importing#import-from-github)?

Currently I can't see any working example with Vite.js.

Thanks for any answer.

**Comment by ghost** [comment]
Hi! Do you have a similar issue as me? https://github.com/codesandbox/codesandbox-client/issues/4670

**Comment by andrasbacsai** [comment]
Hey, 

You can use our SSE containers for vite.

Example with Vue 3: https://codesandbox.io/s/vue3-vite-template-1uvnt?file=/package.json
